1175/en.md

```diff
@@ -0,0 +1,138 @@
+# LOJ 1175 - Jane and the Frost Giants
+
+## Summary
+
+**Input:**
+
+A 2D grid representing the maze, where each square is either empty, blocked by an obstacle, or contains fire lit by the *Frost Giants*, except one cell which is the starting location of *Jane* in the maze. Both *Jane* and the fire take one minute to move to an adjacent cell (horizontally or vertically).
+
+**Output:**
+
+A boolean value indicating whether *Jane* can escape the maze before the fire reaches her. If *Jane* can escape, the minimum number of minutes it takes for her to escape from the maze.
+
+**Note:** *Jane* can escape from any square that borders the edge of the maze.
+
+## Prerequisite
+**BFS** (Bredth First Search): https://cp-algorithms.com/graph/breadth-first-search.html
+
+## Solution
+If we forget about the movable fire cells that grows every minute, then the problem reduces to calculating how many minutes it takes to reach a cell without going through an obstacle, which can be done easily with BFS alone.
+
+ However, taking the fire cells into account, we can think of them as movable osbtacles that expand to the adjacent cells every minute causing *Jane* not to move through them. Before Jane moves to any of her adjacent cells, the fire cells must move to their adjacent cells, because Jane can only move through that cell if the fire hasn't lit up in that very moment. 
```

Comment:
Elaborate this a bit, specially on how you are dealing with fire.

Reply:
I tried my best. What do you think?